test/src/dao/RestDAO.js

```diff
@@ -100,13 +100,14 @@ describe('RestDAO', function() {
             } catch (err) {
               return Promise.resolve(createResponse({ status: 500 }));
             }
+            var sink = data.sink || this.ArraySink.create();
```

Comment:
The current implementation only sends a sink param if a serializable sink was passed to `select()` on the client.